git-workflow.md

```diff
@@ -38,7 +40,9 @@ In GitHub, locate the Kyma repository you want to work on and click the **Fork**
 
 2. Clone the fork to your local machine.
 
-To introduce changes locally, clone the forked repository on your local machine. On GitHub repository's main page, select the **Clone and download** option to copy the fork's address. Use either the terminal and the `git clone https://github.com/{your-username}/{your-fork-name}.git` command or the Git GUI to clone the forked repository.  
+To introduce changes locally, clone the forked repository on your local machine. On GitHub repository's main page, select the **Clone and download** option to copy the fork's HTTPS address. Use either the terminal and the `git clone https://github.com/{your-username}/{your-fork-name}.git` command or the Git GUI to clone the forked repository.  
+
+> **NOTE:** When you have an SSH key added to your GitHub account, use the SSH address to clone the forked repository.  
```

Comment:
I think the note should say: 

> To use SSH to work with the repository, generate and add an SSH key to your account. See the [GitHub guidelines](https://help.github.com/articles/checking-for-existing-ssh-keys/#platform-mac) for details.

Reply:
No problem, I can rephrase that.

Reply:
As discussed, I have simplified the content a bit.